kratos/python_scripts/restart_utility.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, model_part, settings):
         default_settings = KratosMultiphysics.Parameters("""
         {
             "input_filename"                 : "",
-            "io_foldername"                  : "",
+            "io_path"                        : "",
```

Comment:
here I guess the name is ok, although it is again not consistent :/

Reply:
See above, but we could have the output_path for the save_restart and the io_path for the general restart_utility. I would be in favor of io_path for both, but the main importance is we change from folder_name as that is confusing.

Reply:
as I said above I would like it to be 100% consistent, at least for the output processes (which includes the restart process)

the restart utility is used differently, hence there I would accept a different name

For me as it is now is ok

Reply:
For the RestartUtility the PathLoad and PathSave use the same, so I will revert to using output_path.